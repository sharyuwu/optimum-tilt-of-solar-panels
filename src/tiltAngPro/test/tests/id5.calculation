id5_1
23.250943
id5_2
-23.250943
id5_3
-69.250943
id5_4
69.250943