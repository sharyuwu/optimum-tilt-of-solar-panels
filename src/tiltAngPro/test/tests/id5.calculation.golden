.calculation
id5_1
Input 0.9738212193883772 Output 0.9738212 Relative Erros = 1.9909586201904972e-8
id5_2
Input 0.9738212193883772 Output 0.9738212 Relative Erros = 1.9909586201904972e-8
id5_3
Input 0.5786890162786223 Output 0.5786897 Relative Erros = 1.1814991309755385e-6
id5_4
Input 0.5786890162786223 Output 0.5786897 Relative Erros = 1.1814991309755385e-6
